src/compass/utils/validate_product.py

```diff
@@ -210,12 +212,12 @@ def _compare_complex_slc_rasters(file_ref, file_sec, pols):
     pols: list[str]
         List of polarizations of rasters to compare
     """
-    grid_path = f'{DATA_ROOT}/CSLC/grids'
+    data_path = DATA_PATH
     with h5py.File(file_ref, 'r') as h_ref, h5py.File(file_sec, 'r') as h_sec:
         for pol in pols:
             # Retrieve SLC raster from ref and sec HDF5
-            slc_ref = h_ref[f'{grid_path}/{pol}']
-            slc_sec = h_sec[f'{grid_path}/{pol}']
+            slc_ref = h_ref[f'{data_path}/{pol}']
+            slc_sec = h_sec[f'{data_path}/{pol}']
```

Comment:
```suggestion
            slc_ref = h_ref[f'{DATA_PATH}/{pol}']
            slc_sec = h_sec[f'{DATA_PATH}/{pol}']
```
Can use imported value directly.